package.json

```diff
@@ -642,31 +647,31 @@
         "@types/shelljs": "0.8.12",
         "@types/vscode": "^1.62.0",
         "@types/webpack": "^4.41.26",
+        "@vscode/test-cli": "0.0.10",
         "@vscode/test-electron": "^2.4.1",
-        "mocha-junit-reporter": "^2.2.1",
-        "mocha-multi-reporters": "^1.5.1",
         "glob": "^7.2.3",
         "glob-promise": "^4.2.2",
         "gulp": "^4.0.2",
         "mocha": "^10.2.0",
+        "mocha-junit-reporter": "^2.2.1",
+        "mocha-multi-reporters": "^1.5.1",
         "nuget": "^2.0.2",
         "recursive-readdir": "^2.2.3",
         "rimraf": "3.0.2",
         "shelljs": "^0.8.5",
         "source-map-support": "^0.5.21",
         "ts-node": "^10.9.1",
-        "typescript": "^4.5.5",
-        "vsce": "^1.85.0",
+        "typescript": "^3.8.3",
```

Comment:
Unusual to see a change to lower version. Assume this is intentional.

Reply:
Yes, had to do that temporarily.  Will upgrade it again later.  Thanks!